src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -77,6 +79,11 @@ internal IEnumerable<IKeyValueAdapter> Adapters
             set => _adapters = value?.ToList();
         }
 
+        /// <summary>
+        /// A collection of user defined functions that transform each <see cref="ConfigurationSetting"/>.
+        /// </summary>
+        internal IEnumerable<Func<ConfigurationSetting, ValueTask<ConfigurationSetting>>> UserDefinedMappers => _mappers;
```

Comment:
Are there system defined mappers?

Reply:
I think I just overlooked this, will change it to "Mappers"